avocado_vt/options.py

```diff
@@ -152,6 +152,11 @@ def _process_qemu_accel(self):
             self.cartesian_parser.assign("disable_kvm", "yes")
 
     def _process_bridge_mode(self):
+        if self.options.vt_type == 'spice':
+            # Test provider spice wants to specify network parameters in
+            # cartesian config.
+            logging.info("Ignoring predefined network setting for tp-spice.")
+            return
```

Comment:
How about do it like below?

if not self.options.vt_config and self.options.vt_type != 'spice':
       .............
else:
       logging.info("Config provided, ignoring %s", nettype_setting)